system wide lib interfere with radium external

On debian bullseye, when `libglm-dev` is installed, then link fail for globjects.
```
[ 84%] Performing build step for 'globjects'
[  1%] Linking CXX shared library ../../libglobjects.so
/usr/bin/ld: cannot find -lglm
```

to reproduce:
```
sudo apt install libglm-dev
```
cmake configuration fail
```
sudo apt remove --purge libglm-dev
```
cmake configuration pass

So it seems that the library search pass is not set correctly during configuration of globjects regarding glm

--- a/external/Engine/CMakeLists.txt
+++ b/external/Engine/CMakeLists.txt
@@ -26,6 +26,7 @@ if (NOT DEFINED glm_DIR)
         ${RADIUM_EXTERNAL_CMAKE_OPTIONS}
         -DGLM_TEST_ENABLE=OFF
         -DBUILD_STATIC_LIBS=OFF
+        -DCMAKE_INSTALL_LIBDIR=lib
         )
 
     set(glm_DIR ${CMAKE_INSTALL_PREFIX}/lib/cmake/glm)
@@ -53,7 +54,7 @@ if (NOT DEFINED glbinding_DIR)
         -DOPTION_BUILD_EXAMPLES=OFF
         )
 
-    set(glbiniding_DIR ${CMAKE_INSTALL_PREFIX})
+    set(glbinding_DIR ${CMAKE_INSTALL_PREFIX}/share/glbinding)
 else()
     add_custom_target(glbinding)
     StatusMessage("[EngineExternal]" "glbinding"  "${glbinding_DIR}")
@@ -84,6 +85,8 @@ if (NOT DEFINED globjects_DIR)
         -Dglm_DIR=${glm_DIR}
         -DEigen3_DIR=${Eigen3_DIR}
         )
+
+    set(globjects_DIR ${CMAKE_INSTALL_PREFIX}/share/globjects)
 else()
     StatusMessage("[EngineExternal]" "globjects" "${globjects_DIR}")
 endif()
@@ -106,5 +109,4 @@ else()
     message(STATUS  "Installing stb_image.h to ${CMAKE_INSTALL_PREFIX}/include")
     file(COPY ${stb_DIR}/stb_image.h DESTINATION  ${CMAKE_INSTALL_PREFIX}/include/stb/)
     file(COPY ${stb_DIR}/stb_image_write.h DESTINATION ${CMAKE_INSTALL_PREFIX}/include/stb/)
-       
 endif()

--- a/external/Engine/package.cmake
+++ b/external/Engine/package.cmake
@@ -1,6 +1,6 @@
-
+# WARNING : if some paths are changed in the following variables, think to change it also in CMakeLists.txt
 if(NOT DEFINED glm_DIR)
-    set(glm_sub_DIR lib/${CMAKE_LIBRARY_ARCHITECTURE}/cmake/glm/ CACHE INTERNAL "")
+    set(glm_sub_DIR lib/cmake/glm/ CACHE INTERNAL "")
     set(glm_DIR ${CMAKE_INSTALL_PREFIX}/${glm_sub_DIR})
 endif()
 

